### ISSUE TITLE
Add API to subscribe to "blink/identify" events

### ISSUE THREAD

**Issue by shaunmulligan** [balena-io/balena-supervisor#1150]
Users would like to trigger custom scripts on the event a "blink/identify" action is triggered from the dashboard. This would allow custom ways of device identification like audio or flashing images on a LCD screen, etc.

**Comment by roman-mazur** [comment]
It makes sense to consider as a specific use case for a more generic "push notification" feature (similar to push notifications on mobile platforms) when our users have API to deliver generic payload to their containers.

**Comment by shaunmulligan** [comment]
@roman-mazur yeah that would be very cool

**Comment by CameronDiver** [comment]
I agree with @roman-mazur I'd much rather we approach this as a generic by-design feature, that the blink action is a user of, rather than hacking this on top of the blink action.

**Comment by robertgzr** [comment]
should this issue be renamed to be about a more generic "event" source?

**Comment by Zapnologica** [comment]
I would love for us to be able to beep our buzzer or re-act to an identify event.
